tests/integration/test_thevenin_parameterisation.py

```diff
@@ -102,5 +102,5 @@ def get_data(self, model, parameters, x):
                 ),
             ]
         )
-        sim = model.predict(experiment=experiment, inputs=x)
+        sim = model.predict(experiment=experiment, inputs=parameters.as_dict(x))
```

Comment:
This should be able to a list without any issues.
```suggestion
        sim = model.predict(experiment=experiment, inputs=x)
```